File: app/src/main/java/com/incendiary/android/ui/main/MainMvpView.java
package com.incendiary.android.ui.main;

import java.util.List;

import com.incendiary.android.data.model.Ribot;
import com.incendiary.android.ui.base.MvpView;

public interface MainMvpView extends MvpView {

    void showRibots(List<Ribot> ribots);

    void showRibotsEmpty();

    void showError();

}

File: app/src/main/java/com/incendiary/androidboilerplate/features/main/MainMvpView.kt
package com.incendiary.androidboilerplate.features.main

import com.incendiary.androidboilerplate.data.model.Ribot
import com.incendiary.androidboilerplate.features.common.MvpView

interface MainMvpView : MvpView {

	fun showRibots(ribots: List<Ribot>)
	fun showRibotsEmpty()
	fun showError()

}
